draft-ietf-tls-esni.md

```diff
@@ -215,14 +215,19 @@ The ECH configuration is defined by the following `ECHConfig` structure.
     struct {
         HpkeKdfId kdf_id;
         HpkeAeadId aead_id;
-    } ECHCipherSuite;
+    } HpkeSymmetricCipherSuite;
 
     struct {
-        opaque public_name<1..2^16-1>;
-        HpkePublicKey public_key;
+        uint8 key_id;
         HpkeKemId kem_id;
-        ECHCipherSuite cipher_suites<4..2^16-4>;
+        HpkePublicKey public_key;
+        HpkeSymmetricCipherSuite cipher_suites<4..2^16-4>;
```

Comment:
IMO it's a bit tidier to have `cipher_suites` immediately precede the `kem_id`.

Reply:
I don't have a strong feeling here. I think the only important bit is that the kem_id precedes the public key.

Reply:
Any change is a change and they are all equally not-free, so minimizing changes is not a goal I would worry about.  In terms of ordering, I prefer what @chris-wood proposed.